pkg/metrics/metrics.go

```diff
@@ -95,6 +100,9 @@ type operationMetricsManager struct {
 
 	// opLatencyMetrics is a Histogram metrics
 	opLatencyMetrics *k8smetrics.HistogramVec
+
+	// opCountMetrics is a Counter metrics
+	opCountMetrics *k8smetrics.CounterVec
```

Comment:
do we really need this? I thought HistogramVec already provides count? given that the metrics are labeled using status, a count can be easily achieved by doing a label = "success"/"error"?

Reply:
Good point, removed the counter and added status to the histogram.